.github/workflows/push.yml

```diff
@@ -32,6 +32,52 @@ env:
   CC: ccache gcc
   CXX: ccache g++
 jobs:
+  ALPINE:
+    if: github.repository == 'php/php-src' || github.event_name == 'pull_request'
+    name: ALPINE_X64_ASAN_UBSAN_DEBUG_ZTS
+    runs-on: ubuntu-22.04
+    container:
+      image: 'alpine:3.20.1'
```

Comment:
I will modernize the push and nightly Alpine jobs in a separate PR.

The latest Alpine version has Clang 20, so we can have at least one job that runs the tailcall VM.

Reply:
Latest stable is 3.22 and it has clang 15-20 https://pkgs.alpinelinux.org/packages?name=clang1%3F&branch=v3.22&repo=&arch=x86_64&origin=&flagged=&maintainer=